pkg/printer/printer_test.go

```diff
@@ -8,25 +8,31 @@ import (
 )
 
 func TestParseFormat(t *testing.T) {
-	testcases := map[string]bool{
-		"plaintext": true,
-		"json":      true,
-		"human":     false,
+	testcases := []struct {
+		rawFormat  string
+		wantFormat Format
+		wantErr    string
+	}{
+		{rawFormat: "plaintext", wantFormat: FormatPlaintext},
+		{rawFormat: "json", wantFormat: FormatJson},
+		{rawFormat: "yaml", wantFormat: FormatYaml},
+		{rawFormat: "", wantFormat: FormatPlaintext},
+		{rawFormat: "human", wantErr: "invalid format"},
 	}
 
-	for name, valid := range testcases {
-		t.Run(name, func(t *testing.T) {
+	for _, tc := range testcases {
+		t.Run(tc.rawFormat, func(t *testing.T) {
 			opts := PrintOptions{
-				RawFormat: name,
+				RawFormat: tc.rawFormat,
 			}
 
 			err := opts.ParseFormat()
-			if valid {
+			if tc.wantErr == "" {
 				require.Nil(t, err)
-				require.Equal(t, name, string(opts.Format))
+				require.Equal(t, tc.wantFormat, opts.Format)
 			} else {
 				require.NotNil(t, err)
```

Comment:
nit - I know this isn't part of this change in the PR, but we probably could change this to require.Error instead?

Reply:
Fixed!